client/index.html.tpl

```diff
@@ -89,7 +89,7 @@
 						<span id="nick"></span>
 						<textarea id="input" class="mousetrap"></textarea>
 						<span id="upload-tooltip" class="tooltipped tooltipped-w tooltipped-no-touch" aria-label="Upload File">
-							<input id="upload-input" type="file" multiple></button>
+							<input id="upload-input" type="file" multiple>
```

Comment:
This should be self-closing then, if not closed by a closing tag.

Reply:
Inputs don't have to.

Reply:
In good ol' XHTML it would have needed to so it can be valid XML, but it doesn't need to be strict XHTML to be valid html5

Reply:
Yeah I saw that on w3c after posting it. I mostly learnt with xhtml and so it's a hangover from that :-P